engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -140,6 +140,27 @@ paths:
         - google_oauth:
             - openid
   '/api/workflows/{version}/{id}/labels':
+    get:
+      summary: Retrieves the current labels for a workflow
+      parameters:
+        - $ref: '#/parameters/versionParam'
+        - $ref: '#/parameters/singleId'
```

Comment:
ToL: is `singleId` the same name used by other lookup endpoints? I've not noticed it before but it's quite possible that it is!

Reply:
But if it is, change the URL format which is currently `id`

Reply:
This was a copy/paste w/ changes from some other entry, so at least one does. I'm just lookingat the thumbnail but I think `singleId` is just a swagger alias

Reply:
Confirming thisi s just an artifact of swagger yaml

Reply:
Just cross reference against the URL about 5 lines up and 👍

Reply:
@cjllanwarne I'm not sure what you're asking and/or saying. Look at the abort/status/logs/etc endpoints, this is the same